src/components/pages/dashboard.tsx

```diff
@@ -418,7 +418,8 @@ const Dashboard = (): any => {
         {t('dashboard.welcome')}, {user?.username}
       </Page.Title>
       <Page.Actions>
-        <Link href="/create" css="text-transform: lowercase;">
+        {/* TODO: Should become /create */}
```

Comment:
You could add a redirect from `/create` to `/event` for now so you can already start using the new path instead of adding a `TODO` (which are often never fixed later)

Reply:
Good point